src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -6940,6 +6925,57 @@ namespace PackagedTerminalHeatPump {
 
 	}
 
+	void
+	SetMinOATCompressor(
+		int const PTUnitNum, // index to furnace
+		std::string const PTUnitName, // name of furnace
+		std::string const cCurrentModuleObject, // type of furnace
+		std::string const CoolingCoilType, // type of cooling coil
+		std::string const CoolingCoilName, // name of cooling coil
+		std::string const HeatingCoilType, // type of heating coil
+		std::string const HeatingCoilName, // name of heating coil
+		bool & ErrorsFound // GetInput logical that errors were found
+	) {
```

Comment:
@mjwitte also has some changes and suggested a new issue to resolve all comments noted here.
 
- unify use of CondenserInletAirTemp between parent and child (both use same information)
- change function call to use int's
- revise input field name? to Minimum Condenser Inlet Air Temperature for Compressor Operation
- others